docs/interviews/tendermint-bft.md

```diff
@@ -0,0 +1,250 @@
+# Interview Transcript with Tendermint core researcher, Zarko Milosevic, by Chjango
```

Comment:
weird that this lives in here?

Reply:
lol

Reply:
yep, if this is published, we can remove it